addons/sourcemod/scripting/shavit-mapchooser.sp

```diff
@@ -45,6 +45,9 @@ int gI_Driver = Driver_unknown;
 Database g_hDatabase;
 char g_cSQLPrefix[32];
 
+//Jacob added this
```

Comment:
don't add "jacob added this" comments everywhere. just include your name (microwavedbunny or something) in the contributor list at the top of the file (line 3)

Reply:
also appending your name to the plugin author section

Reply:
Yeah sorry those comments were so I could find my place.